### DISCUSSION TITLE
스크립트로 네이버 잔여백신 예약중입니다

### DISCUSSION THREAD

**Discussion by mimkim** [kimytsc/covid-rest-vaccine-macro#84] [Q&A]
먼저 원하는 범위 내에 있는 병원 예약창으로 들어가려고 네이버맵에서 범위 지정해서 그 주변에 있는 병원 리스트 빼고있었는데,
![image](https://user-images.githubusercontent.com/88809461/130537392-61151e8b-ed45-4378-af9c-5054fa3822af.png)

원래는 graphql 쭉 누르다보면 목록볼수있는 곳이 한군데 정도는 있었는데 지금 볼수있는곳을 찾을수가 없어요 혹시해서 눌러서 보다보면 저렇게 안뜨구요 

어떻게해야할까요?

**Comment by kimytsc** [comment]
지금 확인해보니 네이버 지도의 api가 `/api/vaccine`으로 변경된 것 같습니다.
